tuf/client/updater.py

```diff
@@ -659,6 +659,10 @@ def __init__(self, repository_name, repository_mirrors):
                                           'targets_path': 'targets',
                                           'confined_target_dirs': ['']}}
 
+      fetcher:
+        A conctrete 'FetcherInterface' implementation. If not provided,
+        tuf.fetcher.RequestsFetcher is used.
```

Comment:
typo on concrete. Also it's lacking a sentence on what fetcher does.

Reply:
Fixed in b0e8acb